data_structures/segment_tree/C++/segment_Tree_with_lazyprop.cpp

```diff
@@ -0,0 +1,138 @@
+#include<bits/stdc++.h>
```

Comment:
Hey @Cryptedknight, thanks for the contribution! 😄

Please take a look at our [rules and guidelines.](https://github.com/pirataspinhal/algorithms/blob/master/CONTRIBUTING.md#2-make-sure-to-that-the-code-youre-adding-is-of-your-own-creation) I apologize, but we cannot accept this file.

Source found on: https://www.geeksforgeeks.org/lazy-propagation-in-segment-tree/